pkg/plugin/system/aspectplugins.go

```diff
@@ -24,10 +24,11 @@ const (
 
 // AspectPlugin represents a plugin entry in the plugins file.
 type AspectPlugin struct {
-	Name       string                 `yaml:"name"`
-	From       string                 `yaml:"from"`
-	LogLevel   string                 `yaml:"log_level"`
-	Properties map[string]interface{} `yaml:"properties"`
+	Name          string                 `yaml:"name"`
+	From          string                 `yaml:"from"`
+	LogLevel      string                 `yaml:"log_level"`
+	PropertiesMap map[string]interface{} `yaml:"properties"`
+	properties    []byte
```

Comment:
How about this? At least they match YAML -> struct field.
```suggestion
	Properties map[string]interface{} `yaml:"properties"`
	propertiesBytes    []byte
```

Reply:
Alternatively, `propertiesRaw`?

Reply:
I like `propertiesBytes`